src/main/kotlin/no/nav/tilleggsstonader/sak/statistikk/vedtak/domene/UtbetalingerDvhV2.kt

```diff
@@ -0,0 +1,55 @@
+package no.nav.tilleggsstonader.sak.statistikk.vedtak.domene
+
+import no.nav.tilleggsstonader.sak.statistikk.vedtak.AndelstypeDvh
+import no.nav.tilleggsstonader.sak.utbetaling.tilkjentytelse.domain.AndelTilkjentYtelse
+import no.nav.tilleggsstonader.sak.utbetaling.tilkjentytelse.domain.TypeAndel
+import no.nav.tilleggsstonader.sak.util.toYearMonth
+import no.nav.tilleggsstonader.sak.vedtak.barnetilsyn.beregning.finnMakssats
+import no.nav.tilleggsstonader.sak.vedtak.domain.InnvilgelseTilsynBarn
+import no.nav.tilleggsstonader.sak.vedtak.domain.Vedtak
+import no.nav.tilleggsstonader.sak.vedtak.domain.VedtakUtil.takeIfType
+import java.time.LocalDate
+
+data class UtbetalingerDvhV2(
+    val fraOgMed: LocalDate,
+    val tilOgMed: LocalDate,
+    val type: AndelstypeDvh,
+    val beløp: Int,
+    val makssats: Int? = null,
+    val erMakssats: Boolean? = null,
+) {
+    data class JsonWrapper(
+        val utbetalinger: List<UtbetalingerDvhV2>,
+    )
+
+    companion object {
+        fun fraDomene(ytelser: List<AndelTilkjentYtelse>, vedtak: Vedtak?) = JsonWrapper(
+            ytelser.filterNot { it.type == TypeAndel.UGYLDIG }.map { andelTilkjentYtelse ->
+
+                val makssats = andelTilkjentYtelse.finnMakssats(vedtak)
+
+                UtbetalingerDvhV2(
+                    fraOgMed = andelTilkjentYtelse.fom,
+                    tilOgMed = andelTilkjentYtelse.tom,
+                    type = AndelstypeDvh.fraDomene(andelTilkjentYtelse.type),
+                    beløp = andelTilkjentYtelse.beløp,
+                    makssats = makssats,
+                    erMakssats = makssats?.let { makssats <= andelTilkjentYtelse.beløp },
```

Comment:
Føler ikke det blir helt riktig
Beløpet i andelen er beløpet for de dager man er i aktivitet. 

https://github.com/navikt/tilleggsstonader-sak/blob/main/src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/barnetilsyn/beregning/TilsynBarnBeregningService.kt#L136-L139

Hvis man har 5000kr i utgifter, så får man eks kun dekkt 3000kr. Men så er man i aktivitet 2 dager i en måned, og får 200kr i beløp i andel. Men du har fortsatt nått makssatsen på en måte.

Reply:
Dette er fikset nå!